lua/autorun/server/sv_daily_restart.lua

```diff
@@ -171,11 +332,40 @@ local function onAlertTimeout()
         msg = msg .. secondsUntilNextRestart .. " second"
         if ( secondsUntilNextRestart > 1 ) then msg = msg .. "s" end
     end
-    msg = msg .. "!"
+
+    return msg .. "!"
+end
+
+local function onHardAlertTimeout()
+    if canRestartServer() then return restartServer() end
+
+    local secondsUntilNextAlert, secondsUntilNextRestart = getSecondsUntilAlertAndRestart()
+    if secondsUntilNextAlert == nil or secondsUntilNextRestart == nil then return restartServer() end
+
+    local msg = formatAlertMessage( "Restarting server in ", secondsUntilNextRestart )
+    local notifMsg = msg .. "\nThis is a hard restart!\nThe server takes at most 5 minutes to come back online."
```

Comment:
I think we can safely promise 3 minutes here:

```suggestion
    local notifMsg = msg .. "\nThis is a hard restart!\nThe server takes at most 3 minutes to come back online."
```